libraries/AP_SmartAudio/AP_SmartAudio.cpp

```diff
@@ -0,0 +1,947 @@
+#include "AP_SmartAudio.h"
+#include <AP_Math/crc.h>
+#include <GCS_MAVLink/GCS.h>
+
+#define AP_VTX_DEV_MODE     // DEV TESTS ACTIVATE
+#ifdef AP_VTX_DEV_MODE
+# define debug(fmt, args...)	hal.console->printf("\r " fmt "\n", ##args)
+//#define debug(fmt,args...) gcs().send_text(MAV_SEVERITY_DEBUG , fmt, ##args);
+#else
+//# define debug(fmt, args...)	do {} while(0)
+#endif
+
+extern const AP_HAL::HAL &hal;
+const char *TAG="VTX-SMARTAUDIO ";
+
+
+// table of user settable parameters
+const AP_Param::GroupInfo AP_SmartAudio::var_info[] = {
+
+    // @Param: ENABLED
+    // @DisplayName: Enable SmartAudio protocol on VTX
+    // @Description: Enable SmartAudio protocol on VTX
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Advanced
+    AP_GROUPINFO_FLAGS("ENABLED",     1, AP_SmartAudio::configured_smartaudio_params, enabled, 0,AP_PARAM_FLAG_ENABLE),
+
+    // @Param: VERSION
+    // @DisplayName: SmartAudio version
+    // @Description: SmartAudio version
+    // @Values: 0: Version 1,1:Version 2, 3: Version 2.1
```

Comment:
Can we determine this at runtime? Would make for a better user experience

Reply:
Fixed, i remove it